.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 cache:
-  bundler: true
+  bundler: false
```

Comment:
I found that the yarn cache is disabled in f5bf5ebb (`cache.yarn: false`). I suppose the logic would be to save disk space (?) from inefficient cache and maybe I should follow (?).

I can revert this commit and bundler cache will be functional.